### ISSUE TITLE
 `Hmm... it's not here` page displayed for old expenses

### ISSUE THREAD

**Issue by m-natarajan** [Expensify/App#49190]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 9.0.33-1
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** @iwiznia 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1726241308536949

## Action Performed:
1. Open old expense reports from LHN
## Expected Result:
Reports displayed without any issue
## Actual Result:
Redirected to https://staging.new.expensify.com/search/view/3431960930982529 or https://staging.new.expensify.com/search/view/0 (this same IDs every time, no matter the transaction I click) and see `Hmm... it's not here` page
## Workaround:
Unknown
## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

https://github.com/user-attachments/assets/8beadce4-579e-467c-a19f-390a0c4683da

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by CyberAndrii** [comment]
Might be the same root cause as https://github.com/Expensify/App/issues/46576

**Comment by sonialiap** [comment]
I agree, it looks very similar. Closing this in favor of https://github.com/Expensify/App/issues/46576
